policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml

```diff
@@ -5,51 +5,99 @@ metadata:
   amount_unit: /1
   label: Arkansas income tax amount
   reference:
-  - title: Tax Legislation for 2022 Ark. Code § 26-51-201 (a)(1)(A)
-    href: https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-2-imposition-of-tax/section-26-51-201-effective-90-days-after-sine-die-adjournment-individuals-trusts-and-estates
-
-  - title: 2022 Indexed Tax Brackets
-    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2022.pdf#page=1
+  - title: 2014 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2014.pdf#page=1
+  - title: 2015 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2015.pdf#page=1
+  - title: 2019 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2019.pdf#page=1
+  - title: 2020 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2020.pdf#page=1
   - title: 2021 Indexed Tax Brackets
     href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2021.pdf#page=1
+  - title: 2022 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2022.pdf#page=1
   - title: 2023 Indexed Tax Brackets
     href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2023.pdf#page=1
-    
+
 brackets:
   - threshold:
-      2021-01-01: 0
+      2014-01-01: 0
     amount:
-      2021-01-01: 0
+      2014-01-01: 0.009 
+      2019-01-01: 0
+      # one rate for multiple threshold values??
+      uprating:
+        parameter: gov.irs.uprating
+        rounding:
+          type: downwards
+          interval: 100
```

Comment:
The uprating should only be applied to the thresholds not the rates, see example below